misk-hibernate/src/main/kotlin/misk/hibernate/RealTransacter.kt

```diff
@@ -37,6 +83,13 @@ internal class RealTransacter(
     }
   }
 
+  override fun retries(): Transacter = withOptions(options.copy(maxAttempts = 2))
```

Comment:
I don't like that this duplicates the default value in a different place. May add a constant.